cmake/std/PullRequestLinuxGCC8.3.0TestingSettings.cmake

```diff
@@ -17,6 +17,12 @@ set (MPI_EXEC_PRE_NUMPROCS_FLAGS "--bind-to;none" CACHE STRING "Set by default f
 
 set (Trilinos_ENABLE_COMPLEX_DOUBLE ON CACHE BOOL "Set by default for PR testing to exercise complex doubles case")
 
+# Enable C++14 for this build. In the future how we do this will change, but for
+# now we need to use the Kokkos option
+
+set (Kokkos_CXX_STANDARD "14" CACHE STRING "Set C++ standard to C++14")
```

Comment:
change this to CMAKE_CXX_STANDARD instead of Kokkos_CXX_STANDARD and we should be good